addon/components/base/bs-form/element/control/input.js

```diff
@@ -32,7 +32,8 @@ export default Control.extend({
     'step',
     'form',
     'spellcheck',
-    'title'
+    'title',
+    'ariaDescribedby:aria-describedBy'
```

Comment:
I also thought about this, but this does not apply to only that attribute. In fact you could put most attributes to the parent `control` that are shared among all the current controls. But afaik you cannot remove things from `attributeBindings` in a child class that a parent class has added. So I don't know if putting all into the parent is not assuming too much, like what happens when we add a special control that should not have some of those attributes applied? So I thought even if there is some redundancy involved here, we keep the actual control responsible for defining its allowed attributes. But I'm open to other arguments ;)